static/app/components/dropdownMenu/item.tsx

```diff
@@ -52,7 +51,7 @@ export interface MenuItemProps extends MenuListItemProps {
    * Function to call when user selects/clicks/taps on the menu item. The
    * item's key is passed as an argument.
    */
-  onAction?: (key: MenuItemProps['key']) => void;
```

Comment:
It does look like there was a breaking change which let to removing this argument.

`useMenuItem()` now prefers the `onAction` passed in to the item props: https://github.com/adobe/react-spectrum/blob/c904e066c34160803a0aa081e13821d8f67e3e84/packages/%40react-aria/menu/src/useMenuItem.ts#L134-L138

When it uses the `onAction()` from the item props, it does not pass `key`. It doesn't seem necessary to me since the items will already know what their keys are, so I removed it from here and in the areas that passed it.